pkg/api/status_test.go

```diff
@@ -30,7 +30,8 @@ import (
 
 const (
 	// this JSON was generated from the mon_command "status",  ExecuteMonCommand(conn, map[string]interface{}{"prefix": "status"})
```

Comment:
update this comment about where this output was generated from? it wasn't from a mon_command anymore, right?